parameter_files/fates_params_default.cdl

```diff
@@ -12,6 +12,8 @@ dimensions:
 	fates_prt_organs = 6 ;
 	fates_string_length = 60 ;
 	fates_variants = 2 ;
+        fates_hlm_pftno = 14 ;        
```

Comment:
this forces an update to the trait sorting script.  i'll make a PR to your PR branch to do that.